March update first draft (#282)
simplabs/simplabs.github.io: blog/_posts/2019-03-01-march-monthly-update.md

---
layout: article
section: Blog
title: "Simplabs monthly update - March 2019"
author: "Mark Coleman"
github-handle: mrmrcoleman
twitter-handle: mrmrcoleman
topic: simplabs
---

February has been a hive of activity and March looks to be even busier so we're introducing a new monthly update! Read on to find out what we've up to and where you can meet us in March.

<!--break-->

> ##Upcoming Events

Reviewer: ```suggestion
## Upcoming Events
```

The space is needed as otherwise the formatting breaks

[EmberConf 2019](https://emberconf.com/) in Portland is only a few weeks away and we'll be both sponsoring and attending in force, plus our own [Jessica Jordan](https://twitter.com/jjordan_dev) will be giving a talk about [Crafting Web Comics in Ember](https://emberconf.com/speakers.html#jessica-jordan). If you're attending, be sure to come and say hello!

Also, Jessica is co-organizing Ember.js Berlin on March 5th. Full lineup with [Rami Al-Rihawi](https://twitter.com/rrihawi_) and [Isaac Ezer](https://twitter.com/isaacezer) [**here**](https://www.meetup.com/Ember-js-Berlin/events/258984499/).

> [Chris Manson](https://www.twitter.com/real_ate) has a busy month ahead over in Dublin. On March 5th he's co-organizing the DublinJS meetup at [Workday](https://twitter.com/workday). Full line up with [Gabriel Costa ](https://twitter.com/gcgoncalves) and [Luciano Mammino](https://twitter.com/loige) [**here**](https://www.meetup.com/DublinJS/events/fbllfpyzfbhb/).

Reviewer 2: ```suggestion
[Chris Manson](https://www.twitter.com/real_ate) has a busy month ahead over in Dublin. On March 5th he's co-organizing the DublinJS meetup at [Workday](https://twitter.com/workday). Full line up with [Gabriel Costa](https://twitter.com/gcgoncalves) and [Luciano Mammino](https://twitter.com/loige) [**here**](https://www.meetup.com/DublinJS/events/fbllfpyzfbhb/).
```

Then on March 28th Chris will be reviving the Ember.js Dublin meetup "in a big way!" with an Ember Octane Workshop at the [Intercom](https://twitter.com/intercom) offices. Register [here](https://www.meetup.com/emberjsdublin/events/259356879/).

> ##Looking back

Reviewer: ```suggestion
## Looking back
```

February was full of events and some interesting work spent on our 20% days. Here's a run down.

### Ember.js film premiere!

We were very pleased that our CEO Marco Otte-Witte was featured alongside Ember.js co-creators [Yahuda Katz](https://twitter.com/wycats) and [Tom Dale](https://twitter.com/tomdale) in the HoneyPot's new film ["Ember.js - The Documentary"](https://www.youtube.com/watch?v=Cvz-9ccflKQ)!

The film was premiered at meetups across Europe and we'd like to thank [Honeypot](https://twitter.com/honeypotio) for putting it together!

### 20% days - Sponsoring Ricardo Mendes - Ember.js Core Team

> At Simplabs we follow Google's 20% rule where each person gets to 1 day per week on a project of their choice.

Reviewer: ```suggestion
At simplabs we follow Google's 20% rule where each person gets to 1 day per week for open source, supporting the projects that we leverage in our client projects.
```

_"simplabs"_ should be lowercase - not sure the other change is well written english but I think we should make clear that a good part of these 20% of our time goes into supporting the projects we rely on. Also to be fair, 20% is not totally of everyone's choice - doing something that's totally unrelated to what simplabs is doing would at least require upfront discussion and might then be deemed not appropriate for 20% time.

> In January and February we were very pleased to sponsor [Ricardo Mendes](https://twitter.com/locks) from the Ember.js Core Team. Chris Manson spent his 20% time pairing with Ricardo on [Guidemaker](https://github.com/ember-learn/guidemaker-ember-template) to add features to help the Ember Octane documentaton process and released a 1.0 of Guidemaker!

Reviewer 2: ```suggestion
In January and February we were very pleased to sponsor [Ricardo Mendes](https://twitter.com/locks) from the Ember.js Core Team. Chris Manson spent his 20% time pairing with Ricardo on [Guidemaker](https://github.com/empress/guidemaker) to add features to help the Ember Octane documentation process and released a 1.0 of Guidemaker!
```

### Events

_We're always interested in new speakers for the various meetups we're involved in. If you'd like to give a talk, [get in touch!](https://simplabs.com/contact/index.html)_

February was a busy month for events. Below are some of the highlights.

Jessica gave a talk at [c't webdev](https://twitter.com/ct_webdev) where she spoke about [Crafting Web Comics in Ember](https://ctwebdev.de/programm.html#slot-21).

Marco ran the [February Ember.js meetup](https://www.meetup.com/Ember-js-Munich/events/258726028/) featuring [Toon Verwaest](https://twitter.com/tverwaes) and [Michael Klein](https://twitter.com/LevelbossMike).

> And last but my no means least, [Tobias Bieniek](https://twitter.com/tobiasbieniek) asked the question [_Help! How do you let others know what’s going on when you’re 8000 feet up in the air in a plane without an engine?!_](http://bangbangcon.com/west/speakers/#tobias-bieniek) at [!!con](https://twitter.com/bangbangcon)!

Reviewer 3: ```suggestion
And last but my no means least, [Tobias Bieniek](https://twitter.com/tobiasbieniek) asked the question [_Help! How do you let others know what’s going on when you’re 8000 feet up in the air in a plane without an engine?!_](http://bangbangcon.com/west/speakers/#tobias-bieniek) at [!!Con West](https://twitter.com/bangbangconwest)!
```
